features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/h2.sql

```diff
@@ -1849,6 +1852,10 @@ CREATE TABLE IF NOT EXISTS ENDPOINT_REG_ENTRY (
   SERVICE_CATEGORY VARCHAR(50),
   ENDPOINT_DEFINITION BLOB,
   REG_ID INTEGER NOT NULL,
+  CREATED_TIME TIMESTAMP,
```

Comment:
Isn't CREATED_TIME and UPDATED_TIME set  CURRENT_TIMESTAMP in the script? Or is it handled in the java code?

Reply:
Handled in the java code